genfragments/ThirteenPointSixTeV/Higgs/HH/HHto2G2L2Nu_TuneCP5_13p6TeV_madgraph-pythia8.py

```diff
@@ -0,0 +1,41 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8ConcurrentHadronizerFilter",
+                         maxEventsToPrint = cms.untracked.int32(1),
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         filterEfficiency = cms.untracked.double(1.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         comEnergy = cms.double(13600.),
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        processParameters = cms.vstring(
+            '15:onMode = on', # allow all tau decays. Leptonic and Hadronic 
+            '24:mMin = 0.05', # the lower limit of the allowed mass range generated by the Breit-Wigner (in GeV)
+            '24:onMode = off', # Turn off all W decays 
+            '24:onIfAny = 11 13 15', # Add W->enu, W->munu, W->taunu 
+            '25:onMode = off', # Turn off all H decays 
+            '25:onIfMatch = 22 22', # Add H->gg decay
+            '25:onIfMatch = 24 -24', # Add H->WW decay
```

Comment:
do you need H->ZZ just like 2B2V case

Reply:
A kind reminder on this point. You might want to include Z -> l+l- Z->nunu similar to 2B2V2L2Nu case. 

Also please include 2V in the name for 2G samples, like 2G2V2L2Nu etc.